docs/product/guidelines/javascript.html

```diff
@@ -5,40 +5,37 @@
 ---
 
 <section class="stacks-section">
-
     {% header h2 | Including the Stacks JavaScript %}
 
-    <p class="stacks-copy">While Stacks is first and foremost a CSS library, it also provides commonly used functionality for some components via JavaScript code. Using this functionality is optional. If you only need the styling parts of Stack, you’re free to ignore the provided JavaScript. The converse is not true: The JavaScript components work under the assumption that the Stacks CSS is available.</p>
+    <p class="stacks-copy">While Stacks is first and foremost a CSS library, it also provides commonly used functionality for some components via JavaScript. This functionality is optional. If you only need the styling parts of Stack, you’re free to ignore the provided JavaScript. The converse is not true: The JavaScript components work under the assumption that the Stacks CSS is available.</p>
 
-    <p class="stacks-copy">In order to use the JavaScript components, do the following.</p>
+    <p class="stacks-copy">Stacks JavaScript is currently included within various Stack Overflow projects automatically. If you’re working on a Stack Overflow project, chances are it’s already available for you! If not, reach out to us and we’ll work on getting it setup.</p>
+    <p class="stacks-copy">To include Stacks JavaScript in other projects, do the following.</p>
 
     <ul class="stacks-list">
-        <li>Include the file <code class="stacks-code">dist/js/stacks.min.js</code> in your page. For example, if you use the unpkg CDN, add the tag <code class="stacks-code">&lt;script src="https://unpkg.com/@stackoverflow/stacks/dist/js/stacks.min.js"&gt;&lt;/script&gt;</code> to your HTML. See <a href="/product/guidelines/using-stacks">Using Stacks</a> for more information on unpkg.</li>
+        <li>Include the file <code class="stacks-code">dist/js/stacks.min.js</code> in your page. For example, if you use the unpkg CDN, add the tag <code class="stacks-code">&lt;script src="https://unpkg.com/@stackoverflow/stacks/dist/js/stacks.min.js"&gt;&lt;/script&gt;</code> to your HTML. See <a href="{{ "/product/guidelines/using-stacks" | relative_url }}">Using Stacks</a> for more information on Unpkg and installing Stacks via NPM.</li>
 
-        <li>If you need to support Internet Explorer 11, you also have to include the file <code class="stacks-code">dist/js/stacks.polyfills.min.js</code>. This file has to be included before the main JavaScript file.</li>
+        <li>If you need to support Internet Explorer 11, you also have to include a polyfill <code class="stacks-code">dist/js/stacks.polyfills.min.js</code>. This file has to be included before the main JavaScript file.</li>
```

Comment:
For the record, the official spelling in copy text is "unpkg" with a lowercase U. I have no strong feelings about this though 😊